introduction/E_installation.md

```diff
@@ -45,6 +45,24 @@ We can get node.js from the [download page](https://nodejs.org/download/). When
 
 Mac OS X users can also install node.js via [homebrew](http://brew.sh/).
 
+#### Debian/Ubuntu users
```

Comment:
Since we didn't create a new header for Mac OS X users just above, I don't think we should create a header here.